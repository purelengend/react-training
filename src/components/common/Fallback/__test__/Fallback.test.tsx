import { customRenderer } from '@src/test/test-utils';

import { Fallback } from '..';

describe('Fallback test case', () => {
  it('should render correctly', () => {
    const mockFallback = customRenderer(<Fallback />);

    expect(mockFallback.toJSON()).toMatchSnapshot();
  });
});
