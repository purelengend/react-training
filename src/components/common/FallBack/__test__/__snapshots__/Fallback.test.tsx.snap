// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Fallback test case > should render correctly 1`] = `
<div
  className="d-flex-center _fallback-message_af4e75"
>
  Something went wrong
</div>
`;
